forge/routes/api/template.js

```diff
@@ -28,6 +28,7 @@ module.exports = async function (app) {
      */
     app.get('/:templateId', async (request, reply) => {
         const template = await app.db.models.ProjectTemplate.byId(request.params.templateId)
+        console.log(template)
```

Comment:
```suggestion
```

Reply:
Doh, missed one

Reply:
Err, you've merged this before this was fixed...

Reply:
I am as disappointed about it as you are.